### PR TITLE
Fixed some bugs. Positioned things better with styling and added some…

### PR DESCRIPTION
**Pull request by henninghaakonsen**
… logic.

### CODE REVIEW

File: app/src/api/api.js [hunk 1]
```diff
@@ -48,10 +48,20 @@ export const replyMessage = (message, messageConversationId) =>
         throw error;
       });
 
+export const deleteMessageConversation = messageConversationId =>
+  getD2Instance()
+    .then(instance =>
+      instance.Api.getApi().delete(`messageConversations/${messageConversationId}`))
+    .then(result => result)
```
Comment (varl) [inline app/src/api/api.js]: Why did you add all these lines?

File: app/src/api/api.js [hunk 2]
```diff
@@ -48,10 +48,20 @@ export const replyMessage = (message, messageConversationId) =>
...
+    .catch(error => {
+      throw error;
```
Comment (varl) [inline app/src/api/api.js]: You should wrap your error in a new error when you re-throw to maintain the stacktrace.

`throw new Error(error)`

File: app/src/api/api.js (hunk 1 above)
Comment (henninghaakonsen) [reply app/src/api/api.js]: Changed from multi-delete. The api does not support multi-delete, but I will probably create some epic to handle multi-selection delete at some stage.
